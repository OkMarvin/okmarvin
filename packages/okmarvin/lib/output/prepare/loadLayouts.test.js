const loadLayouts = require('./loadLayouts')
test('returns default value', () => {
  const cb = jest.fn()
  loadLayouts({ root: __dirname, site: { layoutHierarchy: {} } }, cb)
  expect(cb).toBeCalledWith(null, {
    layoutHash: [],
    layouts: Object.create(null)
  })
})
test('returns layouts from fixture', done => {
  const cb = jest.fn((_err, data) => {
    expect(data).toMatchSnapshot()
    done()
  })
  loadLayouts(
    {
      root: __dirname,
      site: { layoutHierarchy: { 'index.js': ['index.js'] } }
    },
    cb
  )
})
