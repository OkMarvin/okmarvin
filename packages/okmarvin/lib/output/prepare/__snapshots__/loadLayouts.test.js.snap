// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`returns layouts from fixture 1`] = `
Object {
  "layoutHash": Array [
    "2f1ef133c3a93568ad886f68a3b98332",
  ],
  "layouts": Object {
    "index.js": [Function],
  },
}
`;
